jina/_docarray.py

```diff
@@ -0,0 +1,13 @@
+try:
+    from docarray import BaseDocument, DocumentArray
+    from docarray.documents.legacy import Document
+    from docarray.documents.legacy import DocumentArray as LegacyDocumentArray
```

Comment:
I still do not understand, why is `legacy` required for the new `DocumentArray`?